### ISSUE TITLE
[Feature Request] Allow for Additional User Fields

### ISSUE THREAD

**Issue by only1alive** [mRemoteNG/mRemoteNG#388]
I would like to request the ability to add additional **User Fields** under the Miscellaneous section. I currently use the User Field to pass arguments to each of my connections upon connecting. These fields are unique to each connection, and would not make sense to configure as an external tool.

I would request being able to add as many (or few) fields as I would like (up to a reasonable number)

**Comment by kmscode** [comment]
Reference: https://mremoteng.atlassian.net/browse/MR-933

**Comment by fotatsos** [comment]
"I currently use the User Field to pass arguments to each of my connections upon connecting."

can you please explain how to do that ?I need all my connections to get the hostname from the  user field and i just cant understand how to do that

**Comment by dimers** [comment]
I would like to second this feature request / enhancement for the addition of more **User Fields** under the Miscellaneous section. I also want to be able to add as many User Fields as I would like up to a reasonable number and to be able to label them as I see fit or have them created with a unique label such as **User Fields B**. Also I'd like the extra User Fields I want to be applied to any new Connection I create and ideally I can define and control this behavior via global app settings in some manner.

**Comment by simon1tan** [comment]
Yes, this would be very nice for use with as a parameter to external tools. Please.

**Comment by robertpopa22** [comment]
**Fix available for testing**

A fix for this issue has been implemented in our community fork (`robertpopa22/mRemoteNG`, branch `main`). A beta build including this fix is available from the fork's Releases page.

Please test and report if this resolves your issue.

---

> *We sincerely appreciate the mRemoteNG maintainers and the wonderful community that has kept this project alive and thriving for over a decade. Your work powers thousands of IT professionals worldwide, including us. Our fork builds entirely on your foundation — we're preparing a consolidated Pull Request to contribute these improvements back to the official project. Thank you for everything you've built.*
>
> *— robertpopa22, community contributor*

**Comment by robertpopa22** [comment]
**Correction:** The download link in my previous comment was broken. Here is the correct one:

**Download pre-built binaries:** https://github.com/robertpopa22/mRemoteNG/releases

Look for the latest **v1.81.0-beta.2** build (x64, x86, ARM64 — both portable and self-contained variants available).

Sorry for the inconvenience!
